### PR TITLE
Feature/update logging

### PR DESCRIPTION
**Pull request by moonso**
Updated init_log and added way to catch streams in log.

### CODE REVIEW

File: chanjo/log.py [hunk 1]
```diff
@@ -1,35 +1,65 @@
...
+    console = logging.StreamHandler()
+    console.setLevel('WARNING')
+    console.setFormatter(formatter)
 
-    # set formatter
-    template = "%(asctime)s\t%(name)s\t%(levelname)s\t%(message)s"
-    formatter = logging.Formatter(template)
-    handler.setFormatter(formatter)
+    if filename:
+        file_handler = logging.FileHandler(filename, encoding='utf-8')
```
Comment (robinandeer) [inline chanjo/log.py]: Super :+1:

File: chanjo/__init__.py [hunk 2]
```diff
@@ -8,6 +8,10 @@
 :copyright: (c) 2014 by Robin Andeer
 :licence: MIT, see LICENCE for more details
 """
+
+import logging
+logger = logging.getLogger(__name__)
```
Comment (robinandeer) [inline chanjo/__init__.py]: This is my only question really

Why not make this a root logger (``logging.getLogger()``) which will then automatically format loggers in the external dependency packages?

Comment (moonso) [inline chanjo/__init__.py]: Well I don't have any argument against that:)

Comment (moonso) [inline chanjo/__init__.py]: Fixed in new commit!

**Comment by robinandeer** [comment]
:shipit:

**Pull request merged**